scalableminds/webknossos
# Update prettier

This updates prettier which allows us to define a prettierrc file. As an example, this is useful for using an IDE prettier package without duplicating the config.
Additionally, I added an am-i-pretty check to the CI tests so that unpretty code cannot be merged. Since this was the case, I also executed prettier and made the changes part of this PR. Therefore, you may want to review by commit.

### Steps to test:
- run yarn install, yarn pretty, and yarn am-i-pretty and check for appropriate results

------
